### PR TITLE
Force interface functions as external (0.5.0)

### PR DESCRIPTION
**Pull request by axic**
Part of #2330.

### CODE REVIEW

File: test/libsolidity/SolidityNameAndTypeResolution.cpp [hunk 1]
```diff
@@ -5879,6 +5879,28 @@ BOOST_AUTO_TEST_CASE(interface_function_bodies)
...
+	)";
+	success(text);
+}
+
+BOOST_AUTO_TEST_CASE(interface_function_public)
+{
+	char const* text = R"(
+		pragma experimental "v0.5.0";
+		interface I {
+			function f() private;
```
Comment (axic) [inline test/libsolidity/SolidityNameAndTypeResolution.cpp]: Must be `public`.

**Comment by chriseth** [comment]
Hm, I would have said that no visibility specifier is allowed and it is external by default, but I'm also fine with this.

**Review by chriseth — APPROVED** [review]

**Comment by axic** [comment]
@chriseth that question is #2330, but no decisions yet :)

**Comment by chriseth** [comment]
Ok, having thought about it again: People don't know what the defaults are, especially not if there is no obvious default, so probably better to force a keyword.

**Pull request merged**